active_contracts.json

```diff
@@ -18,7 +18,7 @@
     "DAOFactory": "0xd52CA6BbfC11627aF9aC3496CAb975ae20E70A81",
     "Registry": "0x2B8C4DD137104d1E869105cd0106e7D9EF955BfE",
     "TokenFactory": "0xE0b582144d9347e849Ab91780f904d4002c55E05",
-    "AllowlistVotingSetup": "0xD39cE95D1e29d4EA9F4E8813d73E68F2F720377a",
+    "AddresslistVotingSetup": "0xD39cE95D1e29d4EA9F4E8813d73E68F2F720377a",
```

Comment:
Please don't rename it here. the address deployed is stil AllowlistVotingSetup and not AddresslistVotingSetup

Reply:
Done